samples/bluetooth/fast_pair/locator_tag/boards/thingy53_nrf5340_cpuapp.overlay

```diff
@@ -4,15 +4,4 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
 
- / {
-	aliases {
-		pwm-spk = &pwm_spk0;
-	};
-
-	pwmspk {
-		compatible = "pwm-leds";
-		pwm_spk0: pwm-spk-0 {
-			pwms = <&pwm1 0 PWM_MSEC(20) PWM_POLARITY_NORMAL>;
-		};
-	};
-};
+#include "thingy53_nrf5340_cpuapp_common.overlay"
```

Comment:
It should be `*.dtsi` file (DTS include). That would separate naming from board-specific overlays and clearly show the intention here.